lnbits/core/views/fiat_api.py

```diff
@@ -16,3 +19,38 @@
 )
 async def api_test_fiat_provider(provider: str) -> SimpleStatus:
     return await test_connection(provider)
+
+
+@fiat_router.post(
+    "/stripe/terminal/connection_token",
+    status_code=HTTPStatus.OK,
+    dependencies=[Depends(check_admin)],
+)
+async def stripe_terminal_connection_token():
+    wallet = await get_fiat_provider("stripe")
+    if not isinstance(wallet, StripeWallet):
+        raise HTTPException(
+            status_code=500, detail="Stripe wallet/provider not configured"
+        )
+
+    try:
+        location_id = getattr(settings, "stripe_terminal_location_id", None)
```

Comment:
- there is no `stripe_terminal_location_id` on `settings`
- is there only one `stripe_terminal_location_id` per LNbits server?

Reply:
Some crumbs from previous attempts